test/map.rs

```diff
@@ -408,19 +421,19 @@ fn test_idempotent_quickcheck_bug2() {
     assert_eq!(m, m_snapshot);
 }
 
-#[test]
-fn test_nop_on_new_map_should_remain_a_new_map() {
-    let mut map = TMap::new();
-    map.apply(map::Op::Nop);
-    assert_eq!(map, TMap::new());
-}
-
 #[test]
 fn test_op_exchange_same_as_merge_quickcheck1() {
     let op1 = map::Op::Up {
         dot: Dot::new(38, 4),
         key: 216,
-        op: map::Op::Nop,
+        op: map::Op::Up {
+            dot: Dot::new(38, 1),
```

Comment:
the version should be > 4 here?

Reply:
The top level version vector should always have the highest versions for each actor, the only constraint on these sub-Op versions is that they are smaller or equal to the vector clocks of the parent CRDT.

Reply:
so `1` is totally fine here